src/dune_rules/compilation_context.ml

```diff
@@ -175,6 +179,37 @@ let create ~super_context ~scope ~expander ~obj_dir ~modules ~flags
     ; stdlib
     }
   in
+  let melange_js_includes =
+    match melange with
+    | None -> None
+    | Some { Melange.In_context.melange_stanza_dir; target } ->
+      let open Resolve.Memo.O in
+      Some
+        (Command.Args.memo
+           (Resolve.Memo.args
+              (let+ libs = requires_compile in
```

Comment:
should also be the link closure here as well.

Reply:
Using link closure here now as well (note the function was moved from `CC` to `Melange_rules`:

https://github.com/ocaml/dune/blob/370103a3a111de17bcebddd60b23872d134f3208/src/dune_rules/melange_rules.ml#L8